EIPS/eip-4762.md

```diff
@@ -251,7 +251,16 @@ None of:
 
 are warm at the start of a transaction.
 
-Note: When (and only when) calling a system contract _via a system call_, the code chunks and account should not appear in the witness.
+### System contracts
+
+When (and only when) calling a system contract either
+
+ * _via a system call_ or
+ * _to resolve a precompile/opcode_,
+
+ the system contract code chunks and account accesses should not appear in the witness as these should be known/cached in the clients. However any other accesses and all writes should appear in the witness.
```

Comment:
> the system contract code chunks and account accesses should not appear in the witness as these should be known/cached in the clients

What does "account accesses" mean? If a system call or resolving a precompile/opcode reads some state, that should be in the witness right?
The only thing that is known/cached is the system contract code.

Reply:
account accesses: account header accesses

and yes only of system contract's header access is not to be added. if the contract make any accesses while doing normal evm code of some other address it would be included.

may be i should reword it?

Reply:
So this part is then kind of repeating this said earlier?:
>  1. a non-precompile or a system contract is the target of a `*CALL`, `SELFDESTRUCT`, `EXTCODESIZE`, or `EXTCODECOPY` opcode,

If that's the case, I'd remove it just to remove potential confusion if this is referring to something else. But if you think it helps to reinforce, I'm not strongly against it but I'd prefer to put "account **header** accesses" yes.